src/webgpu/api/validation/createView.spec.ts

```diff
@@ -339,3 +340,62 @@ g.test('texture_state')
       texture.createView();
     }, state === 'invalid');
   });
+
+g.test('texture_view_usage')
+  .desc(
+    `Test texture view usage (single, combined, inherited) for every texture format and texture usage`
+  )
+  .params(u =>
+    u //
+      .combine('format', kAllTextureFormats)
+      .combine('textureUsage0', kTextureUsages)
+      .combine('textureUsage1', kTextureUsages)
+      .beginSubcases()
+      .combine('textureViewUsage0', [0, ...kTextureUsages])
+      .combine('textureViewUsage1', [0, ...kTextureUsages])
+  )
+  .beforeAllSubcases(t => {
+    const { format, textureUsage0, textureUsage1 } = t.params;
+    const info = kTextureFormatInfo[format];
+    const textureUsage = textureUsage0 | textureUsage1;
+    t.skipIfTextureFormatNotSupported(format);
+    t.selectDeviceOrSkipTestCase(info.feature);
+    if (textureUsage & GPUTextureUsage.STORAGE_BINDING) {
+      t.skipIfTextureFormatNotUsableAsStorageTexture(format);
+    }
+    if (textureUsage & GPUTextureUsage.RENDER_ATTACHMENT && info.color && !info.colorRender) {
+      t.skip(`Texture with ${format} is not usable as a render attachment`);
```

Comment:
nit: These can be skipped at param building time (using `filter`) rather than here, which reduces the total number of tests.

(Someday the texture format info should depend on the device, but right now we only test base-level devices in most tests.)